service/share/cache_availability.go

```diff
@@ -52,11 +51,9 @@ func (ca *CacheAvailability) Start(ctx context.Context) error {
 	return ca.avail.Start(ctx)
 }
 
-// Stop stops the underlying Availability and closes the CacheAvailability.
+// Stop is an alias for Close to conform to the Availability interface.
```

Comment:
I really don't like this, does "Close" need to exist for some other reason or can we just replace it entirely with "Stop"? Or is this kind of alias okay? 😬

Reply:
I don't like it as well. It doesn't need to exist. See my review below

Reply:
Also, see https://github.com/celestiaorg/celestia-node/pull/891#issuecomment-1179186540